### PR TITLE
Authenticate login

### PR DESCRIPTION
**Pull request by KevinMidboe**
Login credentials are now checked with the server for verification.

### CODE REVIEW

File: GATEWAY/src/pages/login/login.html [hunk 1]
```diff
@@ -10,15 +10,15 @@
...
+      <ion-input type="text" [(ngModel)]="loginInfo.user" name="username" placeholder='TestUser'></ion-input>
     </ion-item>
     <ion-item>
       <ion-label>Host</ion-label>
       <ion-input type="text" [(ngModel)]="loginInfo.host" name="host" placeholder='178.62.99.218'></ion-input>
     </ion-item>
     <ion-item>
       <ion-label>Port</ion-label>
-      <ion-input type="number" [(ngModel)]="loginInfo.port" name="port" placeholder='8080'></ion-input>
+      <ion-input type="number" [(ngModel)]="loginInfo.port" name="port" placeholder='3000'></ion-input>
```
Comment (nornes) [inline GATEWAY/src/pages/login/login.html]: should be 8080

File: GATEWAY/src/pages/login/login.ts [hunk 2]
```diff
@@ -27,29 +28,40 @@ export class LoginPage {
...
-    } else {
-      alert('Invalid login credentials.');
     }
   }
 
   /**
    * Passes the login credidentials from the login form to the connectToServer function.
    */
-  loginForm = (): void => {
+  loginForm() {
```
Comment (nornes) [inline GATEWAY/src/pages/login/login.ts]: why would you do this?

File: GATEWAY/src/pages/virtual-tiles/virtual-tiles.html [hunk 3]
```diff
@@ -42,11 +47,15 @@
...
+          <b>Virtual tiles</b><br>
+          <span style="size: 14;">{{tile.virtualName}}</span>
         </div>
-        <div class="col">
-          <button class="bttn" ion-button small color="positive" (click)="pairTilePopUp(tile)">PAIR ANOTHER</button>
+        <div style="float: right; top: 0;">
+          <b>Physical tiles</b><br>
+          <span style="size: 14;">{{tile.tile._id}}</span>
+        </div>
+        <button class="bttn" ion-button small color="positive" (click)="unpairTile(tile)">Unpair</button>
```
Comment (nornes) [inline GATEWAY/src/pages/virtual-tiles/virtual-tiles.html]: If you want to call it unpair it should call the pairTile with a null value

File: GATEWAY/src/pages/virtual-tiles/virtual-tiles.ts [hunk 4]
```diff
@@ -94,6 +94,13 @@ export class VirtualTilesPage {
     }
   }
 
+  unpairTile = (virtualTile: VirtualTile): void => {
```
Comment (nornes) [inline GATEWAY/src/pages/virtual-tiles/virtual-tiles.ts]: Are you sure we need a specific function for this?

**Review by nornes — CHANGES_REQUESTED** [review]

File: GATEWAY/src/pages/virtual-tiles/virtual-tiles.ts (hunk 4 above)
Comment (KevinMidboe) [reply GATEWAY/src/pages/virtual-tiles/virtual-tiles.ts]: Can the tilesApi function be called from the HTML? And it updates the list so it shows that is connected without refreshing.

File: GATEWAY/src/pages/virtual-tiles/virtual-tiles.html (hunk 3 above)
Comment (KevinMidboe) [inline GATEWAY/src/pages/virtual-tiles/virtual-tiles.html]: I agree.

**Review by nornes — APPROVED** [review]

**Pull request merged**